deployments/examples/predicates/node-affinity-example.yaml

```diff
@@ -37,8 +37,8 @@ spec:
                   - e2e-az2
   containers:
     - name: with-node-affinity
-      image: k8s.gcr.io/pause:2.0
+      image: registry.k8s.io/pause:3.7
       resources:
         requests:
           cpu: "100m"
-          memory: "500M"
\ No newline at end of file
+          memory: "500M"
```

Comment:
as long as this change is not because of any tab char, indent etc we should be good. When users try these examples as is, they should not face any unnecessary issues.